nucypher/policy/conditions/auth/evm.py

```diff
@@ -7,7 +7,7 @@
 from siwe import SiweMessage, VerificationError
 
 
-class Auth:
+class EvmAuth:
```

Comment:
Is this renaming meant to imply that there are other forms of authorization that doe not use EVM-based schemes?

Reply:
Something that is unclear, but possibly OAuth, JWT etc. in the future.